### DISCUSSION TITLE
Weekly Schedule Helper Card

### DISCUSSION THREAD

**Discussion by tomlut** [home-assistant/frontend#13873] [Dashboard]
Would it be possible to add a core card to adjust the weekly schedule helper?

Similar to the helper setup or helper entity settings graphic but with 10 minute precision (or more):

![Untitled](https://user-images.githubusercontent.com/10679300/192173668-bede0866-017b-4da8-839d-6e7782005410.png)

**Comment by lennon101** [comment]
upvote this! came here looking for this option. specifically for the core schedule helper (not custom addons)

**Comment by maikeljkwak** [comment]
Would be great to have a card where standard users could see and modify the scheduler helper. In my case, I am using it for room heating but only the admin account can change the heating schedule in the backend directly

**Comment by BrckedmyRMpro** [comment]
Please,  please,  please! 
I have a schedule for my central heating and I just want a wide friendly way to display that scheduled, maybe like making it a file dashboard.  With a nice clean way to edit abd make changes to the schedule without making it so difficult with the likelihood of accidental deletion while editing the timers.

**Comment by aperezva** [comment]
Me too, vote for it.

**Reply by BrckedmyRMpro** [reply]
I have already made a custom component to fix this.......find my post here: https://community.home-assistant.io/t/how-to-display-a-fullscreen-schedule-view-as-a-dashboard/966741/13?u=angusc
This link is for the installation instructions.  Check the last post in the thread for the latest updated __init__.py which has a back button and the schedule drop down list.

**Comment by tomlut** [comment]
Nope that is nothing like this:

https://user-images.githubusercontent.com/10679300/192173668-bede0866-017b-4da8-839d-6e7782005410.png

**Reply by BrckedmyRMpro** [reply]
It's a custom component to allow the user to simply edit the schedules on their home assistant.  It opens the schedule that you have posted the screenshot and after editing in the easy to use interface,  will be saved and then if you open it again in the HA backend,  will look the same as your screenshot.

**Comment by tomlut** [comment]
Sorry, no. Your interface is not as intuitive as clicking and dragging in the scheduler card.

**Reply by BrckedmyRMpro** [reply]
Maybe you can fix it for your liking then tomlut. I put hours of effort in to solving my problem. If it doesn't work for you then fix your problem.

**Comment by tomlut** [comment]
Mate, you are the one coming here to _my_ feature request spruiking your effort as the solution. Don't get shitty just because I tell you it is not what I am looking for.
